auction-app-frontend/src/services/productService.js

```diff
@@ -30,4 +31,24 @@ const getBidData = (productId) => {
   return getRequest(`/products/${ productId }/bid-details`);
 };
 
-export { getProducts, getProduct, getProductRandom, getProductsByCriteria, getBidData };
+const addProduct = (formData) => {
+  return postRequest("/products", formData);
+};
+
+const findProductsByUserIdAndStatus = (userId, status, page, size) => {
+  console.log("userId", userId);
```

Comment:
I assume you've used it for testing purposes, but please if we aren't using conosle.log or console.error or console.warn for purposes of tracking errors, remove it from the code before pushing it on github